# Thesis_Anna_Laddach_2019_Appendix-A
1. The impact of using different cut-offs for the minimum number of protein core residues.
2. Details of the number of SAVs which localise to different protein regions in the gnomAD
common and rare, COSMIC and Clinvar datasets.
3. A list of pathways annotated by functional cluster ("proliferative", "nucleotide processing"
and "response").
4. Proteins enriched in COSMIC non-driver variants at protein-protein interaction sites.
5. Statistics for comparisons of structural network features between datasets.
6. Numbers of proteins and SAVs which underlie correlations between proteomic/transcriptomic
features and variant enrichment.
7. Pairwise Spearman correlations between all studied proteomics and transcriptomics features.
8. Enrichment of functional pathways by proteomics/transcriptomic features.
9. Properties of trajectories can be represented by 101 snapshots.
